instrumented/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
'Explained' [here](https://github.com/mi-voz-mi-voto/mi-voz-mi-voto/pull/30#discussion_r804866273)